### DISCUSSION TITLE
Optimal threshold?

### DISCUSSION THREAD

**Discussion by samlipworth** [Legana/ampir#18] [Q&A] [answered]
What threshold would you recommend as optimal..? Sorry if I'm being thick but I couldn't see anything on this in your paper I don't think..

**Comment by Legana** [comment, marked as answer]
@samlipworth Thank you for your question. We have touched upon this issue in the supplementary information but I appreciate the difficulty and importance of setting an “optimal threshold”. The optimal threshold is a trade off between being very sure that the positive predictions are true antimicrobial peptides (AMPs) and that you have found all of the AMPs in your dataset. For example, say you want to synthesise 10 peptides for experimental analysis to test antimicrobial activity. You will probably want to set your threshold really high (between .9 and 1, depending on the positive predictions returned) to get the top candidates (and decrease the chance of having false positives). This means you will likely lose many potential AMPs. In an opposite example, say you want to find the most AMPs possible in your dataset (e.g threshold = 0.5) because you are doing a broad scale analysis. In this case, it is much less important that you will have false positives in your dataset because you could potentially do downstream analysis to weed some false positives out. In summary, the higher the threshold, the lower the proportion of false positives. I hope this helps!
